usefulUtilities/airAndWhiteboardDraw/entityResources/entityClientScripts/drawSphereClient.js

```diff
@@ -0,0 +1,743 @@
+                MyAvatar.getJointIndex(dominantHandJoint + "Index3");
+            }
+            if (parentJointIndex === -1) {
+                MyAvatar.getJointIndex(dominantHandJoint);
+                print("ERROR: Falling back to dominant hand joint as index finger tip could not be found");
+            }
+            tablet.tabletShownChanged.connect(_this.tabletShownChanged);
+            HMD.displayModeChanged.connect(_this.displayModeChanged);
+            Window.domainChanged.connect(_this.domainChanged);
+
+            _this.registerControllerMapping();
+            if (HMD.active) {
+                HMD.closeTablet();
+                _this.setUpHMDMode();
+            } else {
+                _this.setUpDesktopMode();
+            }
+            _this.playSound(OPEN_SOUND, OPEN_SOUND_VOLUME, MyAvatar.position, true, false);
+        },
+
+        /* PLAY A SOUND: Plays the specified sound at the position of the user's Avatar using the volume and playback 
+        mode requested. */
+        playSound: function(sound, volume, position, localOnly, loop){
+            if (sound.downloaded) {
+                if (injector) {
+                    injector.stop();
+                    injector = null;
+                }
+                injector = Audio.playSound(sound, {
+                    position: position,
+                    volume: volume,
+                    localOnly: localOnly,
+                    loop: loop
+                });
+            }
+        },
+
+        findWhiteboard: function() {
+            Entities.findEntities(MyAvatar.position, WHITEBOARD_SEARCH_RADIUS_M).forEach(function(entity) {
+                var properties = Entities.getEntityProperties(entity, ['position', "name"]);
+                if (properties.name && properties.name === "Whiteboard") {
+                    if (whiteboard) {
+                        if (Vec3.distance(properties.position, MyAvatar.position) <
+                            Vec3.distance(Entities.getEntityProperties(whiteboard, "position").position, MyAvatar.position)) {
+                            whiteboard = entity;
+                        }
+                    } else {
+                        whiteboard = entity;
+                    }
+                }
+                if (whiteboard) {
+                    whiteboardParts = Entities.getChildrenIDs(whiteboard);
+                    whiteboardParts.push(whiteboard);
+                }
+            });
+        },
+
+        draw: function(onBoard) {
+            if (!readyToDraw) {
+                return;
+            }
+            if (Vec3.distance(previousLinePoint, currentPoint) < MINIMUM_MOVEMENT_TO_DRAW_M ||
+            Vec3.distance(previousLinePoint, currentPoint) > MAXIMUM_MOVEMENT_TO_DRAW_M) {
+                return;
+            }
+            if (onBoard !== wasLastPointOnBoard) { // toggle between on board and air, stop drawing
+                _this.stopDrawing();
+                return;
```

Comment:
don't you need to set `wasLastPointOnBoard` to falsey here?

Reply:
yes...thank you!